app/helpers/results_helper.rb

```diff
@@ -77,6 +106,38 @@ def field_has_results_in_document?(doc, field)
     end
   end
 
+  # renders fields; has an option to submit a list of fields
+  def pick_fields(doc, fields, *list_fields)
```

Comment:
@eshadatta The 'blacklight' way to do this is to delete the entire `_index_default.html.erb` partial because you're just listing fields now and you don't need the custom functionality that was there previously. Then you can delete the `pick_fields` function and let the default blacklight take over